packages/checkout-ui-extensions/src/extension-points.ts

```diff
@@ -222,15 +223,19 @@ export interface ExtensionPoints {
    * header and shipping method options.
    */
   'Checkout::ShippingMethods::RenderBefore': RenderExtension<
-    CheckoutApi & StandardApi<'Checkout::ShippingMethods::RenderBefore'>,
+    ShippingMethodListApi &
```

Comment:
Any reason not to stick to `ShippingMethodsApi` per [tech doc](https://docs.google.com/document/d/1NktKnffnHYk6EezZ5SRwZdt04mQy7I-swd4QssU0qbU/edit#heading=h.xxpf69ehyiln)?